course.py

```diff
@@ -262,16 +265,28 @@ def add_duplicate_course():
     return redirect(url_for('course.course_info', course_id=course_id, year=year))
 
 
+@course_bp.route('/evaluation/<int:evaluation_id>')
+@login_required
```

Comment:
Don't we need to restrict a little more who can show a specific evaluation ? Admin, authoring researcher, course teacher ?

Reply:
I think you reverted the change when merging

Reply:
Nevermind, just saw the check you do after.